core/platform/auth/firebase_auth_services_test.py

```diff
@@ -111,32 +113,86 @@ def install(cls, test):
         'list_users',
         'revoke_refresh_tokens',
         'set_custom_user_claims',
-        'update_user',
         'verify_session_cookie',
     ]
 
     def __init__(self):
         self._users = {}
 
-    def _create_user(self, uid=None, **kwargs):
-        """Adds user to storage if new, otherwise raises an error."""
+    def _create_user(self, uid=None, disabled=False):
+        """Adds user to storage if new, otherwise raises an error.
+
+        Args:
+            uid: str. The unique Firebase account ID for the user. The uid
+                argument cannot be made into a positional argument because the
+                Firebase Admin SDK generates a value when it is None. Generating
+                a value is too complicated for tests, however, so we require
```

Comment:
This isn't a valid reason -- a better reason (if true) would be that the non-test code always passes in a uid so this stub enforces that too. Otherwise we'll need to find a proper way to handle and test the None case.